model/api/src/main/java/org/jboss/aerogear/unifiedpush/utils/KeyUtils.java

```diff
@@ -42,12 +48,29 @@ public static PrivateKey loadPrivateKey(String privateKey) throws  NoSuchAlgorit
     public static PublicKey loadPublicKey(String publicKey) throws NoSuchAlgorithmException, InvalidKeySpecException {
         byte[] decodedPublicKey = Base64Encoder.decode(publicKey);
         KeyFactory keyFactory = KeyFactory.getInstance(ALGORITHM, PROVIDER);
-        ECParameterSpec parameterSpec = ECNamedCurveTable.getParameterSpec(CURVE);
+        ECParameterSpec parameterSpec;
+        parameterSpec = ECNamedCurveTable.getParameterSpec(CURVE);
```

Comment:
Why splitting this to two rows?

Reply:
I think my intellij has drunkenly changed its autoformat rules.